src/multiStore.js

```diff
@@ -2,27 +2,72 @@
  * Creates a multi-store
  *
  * @param {store[]} stores - The stores to sync.
+ * @param {function} [options.onGetError] - A callback to execute if a store errors when getting a value.
+ * @param {function} [options.onSetError] - A callback to execute if a store errors when setting a value.
+ * @param {function} [options.onClearError] - A callback to execute if a store errors when clearing all values.
+ * @param {function} [options.onRemoveError] - A callback to execute if a store errors when removing a value.
  *
  * @returns {object}
  */
-export default stores => {
+export default (
+  stores, 
+  { onGetError, onSetError, onClearError, onRemoveError } = {}
+) => {
+  const formatError = (err, storeIndex) => {
```

Comment:
After thinking about it a bit more, I feel like we shouldn't append the `currentStore` and `nextStore` to the error itself, but simply pass them to the callback:

```js
onGetError(err, key, currentStore, nextStore)
```

The error has nothing to do with it, so let's keep them separate.